File: 1Latihan_B/PHP/DaftarTimSepakBola.php
<?php

class DaftarTimSepakBola
{
    private $nama = "";
    private $negara = "";
    private $tahun_berdiri = "";
    private $pemain = "";

    public function __construct($nama = "", $negara = "", $tahun_berdiri = "", $pemain = "")
    {
        $this->nama = $nama;
        $this->negara = $negara;
        $this->tahun_berdiri = $tahun_berdiri;
        $this->pemain = $pemain;
    }

    public function setNama($nama)
    {
        $this->nama = $nama;
    }

    public function getNama()
    {
        return $this->nama;
    }

    public function setNegara($negara)
    {
        $this->negara = $negara;
    }

    public function getNegara()
    {
        return $this->negara;
    }

    public function setTahunBerdiri($tahun_berdiri)
    {
        $this->tahun_berdiri = $tahun_berdiri;
    }

    public function getTahunBerdiri()
    {
        return $this->tahun_berdiri;
    }

    public function setPemain($pemain)
    {
        $this->pemain = $pemain;
    }

    public function getPemain()
    {
        return $this->pemain;
    }

    public function printHasil()
    {
        return "Nama: " . $this->getNama() . "<br/>" . "Negara: " . $this->getNegara() . "<br/>" . "Tahun Berdiri: " . $this->getTahunBerdiri() . "<br/>" . "Pemain: " . $this->getPemain() . "<br/>";
    }
}

File: 1Latihan_B/C++/DaftarTimSepakbola.cpp
#include <string>
#include <iostream>

using namespace std;

class DaftarTimSepakBola
{
private:
    string nama;
    string negara;
    string tahun_berdiri;
    string pemain;

public:
    DaftarTimSepakBola() {}

    DaftarTimSepakBola(string nama, string negara, string tahun_berdiri, string pemain)
    {
        this->nama = nama;
        this->negara = negara;
        this->tahun_berdiri = tahun_berdiri;
        this->pemain = pemain;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    string getNama()
    {
        return this->nama;
    }

    void setNegara(string negara)
    {
        this->negara = negara;
    }

    string getNegara()
    {
        return this->negara;
    }

    void setTahunBerdiri(string tahun_berdiri)
    {
        this->tahun_berdiri = tahun_berdiri;
    }

    string getTahunBerdiri()
    {
        return this->tahun_berdiri;
    }

    void setPemain(string pemain)
    {
        this->pemain = pemain;
    }

    string getPemain()
    {
        return this->pemain;
    }

    void printHasil()
    {
        cout << "Nama: " << this->nama << endl;
        cout << "Negara: " << this->negara << endl;
        cout << "Tahun Berdiri: " << this->tahun_berdiri << endl;
        cout << "Pemain: " << this->pemain << endl;
        cout << endl;
    }

    ~DaftarTimSepakBola() {}
};